src/components/Button.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = ({
+  wide, color, name, handleClick,
+}) => {
+  const classes = `${color} ${wide ? 'fifty-percent' : 'twenty-five-percent'}`.trim();
```

Comment:
- [ ] Independent of the color the button should also have a border property in CSS.